network-grpc/src/client.rs

```diff
@@ -299,6 +299,15 @@ where
     T::deserialize(&mut buf).map_err(|e| core_client::Error::new(core_client::ErrorKind::Format, e))
 }
 
+fn serialize<T>(x: &T) -> Vec<u8>
```

Comment:
For better consistency with above, perhaps name it `serialize_to_bytes`.

Reply:
hence implementing the chain-core's traits : `Serialize + Deserialize` here.